SublimeQuip.py

```diff
@@ -7,7 +7,7 @@
 from .TabsManager import TabsManager
 from .TabsManager import FILE_TREE_TAB_ID
 
-CACHE_DIRECTORY = sublime.cache_path() + '/QUIP'
+CACHE_DIRECTORY = sublime.cache_path() + "/SublimeQuip"
```

Comment:
revert please, we will rename plugin as Quip

Reply:
5e5ff0b